tiles.js

```diff
@@ -57,7 +57,7 @@ var tiles = [{
     paths: [4, 3, 6, 1, 0, 7, 2, 5]
 }, {
     image: "",
-    paths: [2, 6, 0, 4, 3, 7, 1, 5]
```

Comment:
This is an invalid tile.

[3, 7, 4, 0, 6, 5, 1]

should it be: [3, 4, 7, 0, 1, 6, 5, 2] ?